### PR TITLE
Updated Dockerfile

### PR DESCRIPTION
**Pull request by RahulSurwade08**
## Describe your changes
Updated Dockerfile with Alpine tag and optimized repeating commands.
## Issue ticket number and link
#747
## Checklist before requesting a review
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] New and existing unit tests pass locally with my changes

**Please open the PR against the `dev` branch (Unless the PR contains only documentation changes)**

### CODE REVIEW

File: build/Dockerfile [hunk 1]
```diff
@@ -34,16 +33,16 @@ RUN python build.py
 
 RUN /work/build/ubuntu-latest/kubescape download artifacts -o /work/artifacts
 
-FROM alpine
+FROM alpine:3.16.2
 
 RUN addgroup -S armo && adduser -S armo -G armo
 
-RUN mkdir /home/armo/.kubescape
```
Comment (dwertent) [inline build/Dockerfile:41]: Why did you remove this line?

Comment (RahulSurwade08) [reply build/Dockerfile:41]: COPY command creates the destination if one doesn't exist.

https://docs.docker.com/engine/reference/builder/#copy

**Review by dwertent — APPROVED** [review]
